stew/io2.nim

```diff
@@ -206,6 +230,15 @@ elif defined(posix):
       O_CLOEXEC = cint(0x1000000)
       F_NOCACHE = cint(48)
 
+  type
+    FlockStruct* {.importc: "struct flock", final, pure,
```

Comment:
Nim's posix module already defines this as `Tflock`.

Reply:
Structure definition is incorrect and do not support 64bit platforms.